t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -345,6 +345,18 @@ sub wait_for_element (%args) {
     return $element;
 }
 
+sub wait_for_data_table_entries ($table, $expected_entry_count) {
```

Comment:
Is `$expected_entry_count` actually referring to the expected number of rows in the table?
If yes, then maybe a better phrasing could be used to name this variable.